h/static/styles/partials-v2/_search.scss

```diff
@@ -40,19 +40,38 @@
   margin-right: 30px;
 }
 
-.search-result-list {
+.search-result-container--empty {
+  /* The default height of the search container is calculated to so that it
+     stretches to the bottom of the viewport and no further.
+     The px value here is the height of the stuff above the
+     search-result-container. */
+  height: calc(100vh - 124px);
```

Comment:
I've changed this (and the other one for mobile) so that, on Chrome, there's a 10px gap at the bottom of the viewport. Should be enough to mean no scroll bar in other browsers I think